src/commands/profiles/delete.ts

```diff
@@ -0,0 +1,395 @@
+import * as cosmos from "@azure/cosmos";
+import { Command, flags } from "@oclif/command";
+import * as storage from "azure-storage";
+import cli from "cli-ux";
+import { none, Option, some } from "fp-ts/lib/Option";
+import { FiscalCode } from "italia-ts-commons/lib/strings";
+
+import {
+  config,
+  getCosmosWriteConnection,
+  getStorageConnection
+} from "../../utils/azure";
+import { sequential, sequentialSum } from "../../utils/promise";
+
+type ContainersName =
+  | "profiles"
+  | "messages"
+  | "message-status"
+  | "notifications"
+  | "notification-status"
+  | "sender-services";
+
+type Container = {
+  query: string;
+  containerName: ContainersName;
+  partitionKeySelector: (item: any) => string;
+  queryParamName: string;
+  queryOptions?: cosmos.FeedOptions;
+};
+/**
+ * define a delete operation
+ * a delete operation should have or not related delete operations
+ * a related operation is when we have to retrieve items starting from a given set
+ * .e.g: the delete operation "messages" has the related delete operation "message-status"
+ * because items from "message-status" can be retrieved starting from "message" items
+ */
+type DeleteOp = {
+  relatedOps: ReadonlyArray<Container>;
+  queryParamValue: string;
+  deleteBlobs: boolean;
+} & Container;
+
+export default class ProfileDelete extends Command {
+  public static description = "Delete a profile";
+
+  public static flags = {
+    all: flags.boolean({
+      char: "a",
+      description: "delete items from all containers",
+      required: false,
+      default: false
+    }),
+    profile: flags.boolean({
+      char: "p",
+      description: "delete items from profile container",
```

Comment:
```suggestion
      description: "delete items in the 'profile' container",
```

+ all other descriptions